src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -3,1772 +3,1370 @@
+          <head>include-controls plus matching</head>
+          <p>Controls may also be included using match patterns against their IDs. This is useful
+            because related controls (either in a hierarchy, or together in a group) frequently have
+            related IDs as well.</p>
+          <p>When an import provides the <src>include-controls</src> directive, with a
+              <src>matching</src> child, all controls in the referenced document whose
+              <code>id</code> matches one of the listed <code>pattern</code> values (Glob matching)
+            MUST be explicitly included:</p>
+          <p>If a <code>matching</code> object is provided with no <code>pattern</code>, it MUST be
+            treated as matching nothing.</p>
+
+          <tagging whose="source_profile"><![CDATA[
+include-controls:
+  - matching:
+    - pattern: "ac*" ]]></tagging>
+
+          <p>Using "matching", all non-control children of included controls MUST also be explicitly
+            included.</p>
 
-          </section>
-          <section id="redundant-calls">
-            <head>Redundant calls</head>
-            <p>Calls and matches may be combined in a single import. Their effect is cumulative; any
-              control that is selected by both calling and matching from a given
-                <src>include-controls</src> directive, or by matching more than one pattern, is
-              included once (unless it is also marked for exclusion, see <xref
-                rid="exclude-directive"/>).</p>
-          </section>
         </section>
+
+        <section id="include-child-controls">
+
+          <head>with-child-controls</head>
+          <p>In OSCAL, controls may contain controls. For example, in SP 800-53 many controls are
+            supplemented with control enhancements; in OSCAL these are represented as child controls
+            within parent controls. So parent AC-2 (in a given catalog) has children AC-2(1) through
+            AC-2(13), for example.</p>
+          <p>Child controls can be included by the same mechanism as controls, i.e. by means of an
+            ID call. Alternatively a match can frequently be applied (at least given most ID
+            assignment schemes) to match controls and child controls together.</p>
+
+          <p> A <src>with-child-controls: yes</src> directive on an <src>include-controls</src>
+            indicates that <em>all descendant controls </em> of the explicitly included control MUST
+            also be explicitly included. </p>
+
+          <p> A <src>with-child-controls: no</src> directive on an <src>include-controls</src>
+            indicates that ONLY the explicitly matching control is explicitly included, any
+            descendants are not included. </p>
+
+          <p> If no <src>with-child-controls</src> is provided, the processor MUST consider the
+            directive as being equivalent to one having <src>with-child-controls:no</src>. </p>
+
+        </section>
```

Comment:
This treatment makes it sound like `with-child-controls` is mutually exclusive with `with-ids` and `matching`. This is not true. The construct `with-child-controls` applies to and modifies the behavior of `with-ids` and `matching`. Furthermore, both `with-ids` and `matching` can be used simultaneously. This text should be updated to make this more clear.